channel/channel.go

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package channel
+
+import (
+	"github.com/megaease/easeprobe/notify"
+	"github.com/megaease/easeprobe/probe"
+	log "github.com/sirupsen/logrus"
+	"sync/atomic"
+)
+
+const kind = "channel"
+
+// Channel implements a config for Channel
+type Channel struct {
+	Name      string                    `yaml:"name"`      // unique name
+	Probers   map[string]*probe.Prober  `yaml:"probers"`   // probers
+	Notifiers map[string]*notify.Notify `yaml:"notifiers"` // notifiers
```

Comment:
```suggestion
	Probers   map[string]probe.Prober  `yaml:"probers"`   // probers
	Notifiers map[string]notify.Notify `yaml:"notifiers"` // notifiers
```

as both `probe.Prober` and `notify.Notify` are interfaces.